src/notifications/urls.py

```diff
@@ -14,6 +14,7 @@
     # Simply show the master template.
     path('', TemplateView.as_view(template_name='index.html')),
     path('ref/', include('vng_api_common.urls')),
+    path('logging/', include('notifications.notificatielog.urls', namespace='notificatielog'))
```

Comment:
you don't have to explicitly provide the `namespace` anymore if you use `app_name` in the URLs module